<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_EditSalesOppotunity</class>
 <widget class="QDialog" name="Dialog_EditSalesOppotunity">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>425</width>
    <height>269</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>682</width>
    <height>269</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QComboBox" name="comboBox_material_type"/>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Уверенность в показаниях</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Ожидаемая периодичность</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Ожидаемый объём заказа</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Вероятность успеха</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Первая возможная отгрузка</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Группа товаров</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QSlider" name="horizontalSlider_surelevel">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QSlider" name="horizontalSlider_succprob">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>между</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="dateEdit_start_early"/>
       </item>
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>и</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="dateEdit_start_late"/>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="5" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>каждые</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_expected_periodicity"/>
       </item>
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>к. дн.</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="4" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLineEdit" name="lineEdit_expected_qtty"/>
       </item>
       <item>
        <widget class="QLabel" name="label_meas_unit">
         <property name="text">
          <string>Ед. Изм.</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="6" column="1">
      <widget class="QCheckBox" name="checkBox_instock_promise">
       <property name="text">
        <string>Обещаем держать на складе</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QPushButton" name="pushButton_changedates_to_m">
         <property name="text">
          <string>месяц</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_changedates_to_q">
         <property name="text">
          <string>квартал</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_changedates_to_hy">
         <property name="text">
          <string>полгода</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_changedates_to_y">
         <property name="text">
          <string>год</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog_EditSalesOppotunity</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog_EditSalesOppotunity</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
